db/migrate/20230720165939_add_devise_to_users.rb

```diff
@@ -21,11 +21,11 @@ def self.up
       # t.string   :current_sign_in_ip
       # t.string   :last_sign_in_ip
 
-      ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      # Confirmable
+      t.string   :confirmation_token
+      t.datetime :confirmed_at
+      t.datetime :confirmation_sent_at
+      t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
Why did you uncomment these lines?

Reply:
@shasherazi  I thought it was a requirement confirming a user. Since it was for the last Rails projects. If it is just an additional feature, then I can comment it out. Maybe if we have time we can add that feature? Anyway, as long as we didn't add confirmable on the user model these changes will not affect the registration process.

Reply:
Yeah, it sounds reasonable. Ok keep these and we might figure out an actual mailer at the end of the project.